packages/instant/src/constants.ts

```diff
@@ -16,6 +16,9 @@ export const BUY_QUOTE_UPDATE_INTERVAL_TIME_MS = ONE_SECOND_MS * 15;
 export const DEFAULT_GAS_PRICE = GWEI_IN_WEI.mul(6);
 export const DEFAULT_ESTIMATED_TRANSACTION_TIME_MS = ONE_MINUTE_MS * 2;
 export const ETH_GAS_STATION_API_BASE_URL = 'https://ethgasstation.info';
+export const ANALYTICS_ENABLED = process.env.NODE_ENV === 'production' || process.env.ENABLE_HEAP;
+export const HEAP_ANALYTICS_DEVELOPMENT_APP_ID = '507265531';
```

Comment:
can we move these into our build process? If some one forks this, they could be logging garbage into our events right?

Reply:
Per slack convo, I will move these into ENV vars